### ISSUE TITLE
Missing Type hint when parent class prevents one from being added.

### ISSUE THREAD

**Issue by mikebronner** [slevomat/coding-standard#1175]
I am receiving the following error from PHPCS:

```
SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingAnyTypeHint
```

I would like to re-raise the issue brought up in #904. It is totally possible to inspect parents using reflection. This would be a good thing to add, as these are false positives, and greatly diminishes the effectiveness the linter has (people will start ignoring it because of false positives).

I would like to ask if this could be reconsidered. Thanks :)

**Comment by kukulich** [comment]
Duplicate of #904

**Comment by mikebronner** [comment]
@kukulich I know it's a duplicate, but since #904 is locked, I can't comment on it.
